### PR TITLE
Updating tool chromeister to v1.3.c

### PR DESCRIPTION
**Pull request by estebanpw**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/chromeister/chromeister.xml [hunk 1]
```diff
@@ -44,14 +53,24 @@
         </param>
         <param name="diffuse" type="integer" value="4" min="1" max="4" label="Diffuse value" help="Level of the heuristic subsampling employed. Change to 1 or 2 if no similarity is found"/>
         <param name="grid" type="boolean" truevalue="grid" falsevalue="" checked="true" label="Add grid to plot for multi-fasta data sets" help="Do not use grid if your multi-fasta contains more than a hundred sequences (approximately)"/>
+        <conditional name="options">
```
Comment (bgruening) [inline tools/chromeister/chromeister.xml:56]: this conditional is not needed, you can simply use the select :)

Comment (estebanpw) [reply tools/chromeister/chromeister.xml:56]: but what happens then in line 34? here:
`#if str($options['selection_mode']) == "pngevents":`

or is `options` defined somewhere else?

Comment (bernt-matthias) [reply tools/chromeister/chromeister.xml:56]: Or even a boolean is possibe. 

Then it is `#if $selection_mode`.

Comment (estebanpw) [reply tools/chromeister/chromeister.xml:56]: Can I use the boolean later in a filter? i.e. how can I use it in the following?:
```
<data name="output_events_png" format="png" label="${tool.name} on ${on_string}: Detected events plot">
   <filter> options['selection_mode'] == 'pngevents'</filter>
</data>
```

Comment (bernt-matthias) [reply tools/chromeister/chromeister.xml:56]: Sure

```
<data name="output_events_png" format="png" label="${tool.name} on ${on_string}: Detected events plot">
   <filter>selection_mode</filter>
</data>
```

**Review by bernt-matthias — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Pull request merged**